synapse/storage/databases/main/room.py

```diff
@@ -1320,6 +1320,101 @@ async def add_event_report(
             desc="add_event_report",
         )
 
+    async def get_event_reports_paginate(
+        self,
+        start: int,
+        limit: int,
+        direction: str = "b",
+        user_id: str = None,
+        room_id: str = None,
```

Comment:
Since these arguments can be optional, their type should be `Optional[str]` instead of `str`.